scanpipe/models.py

```diff
@@ -3964,6 +3967,103 @@ def as_spdx(self):
         )
 
 
+class DiscoveredPackageScore(UUIDPKModel, PackageScoreMixin):
+    def __str__(self):
+        return self.score or str(self.uuid)
+
```

Comment:
Follow the conventions used across the existing Models:
1. Fields
2. class Meta
3. __str__